### ISSUE TITLE
multiple default reviewers not working as expected

### ISSUE THREAD

**Issue by PaulHofma** [timvink/doing-cli#31]
Having a space-separated list of reviewers is not giving me the expected results, instead resulting in errors of the form:

```az repos pr create --repository '[...]' --draft 'true' --work-items '[...]' --source-branch '[...]' --title 'testissue' --project '[...]' --organization '[http://...] '
--transition-work-items 'true' --delete-source-branch 'true' --auto-complete 'true' --reviewers 'Name.One@Company.com Name.Two@Company.com My.Name@Company.com'
b'ERROR: Could not resolve identity: Name.One@Company.com Name.Two@Company.com My.Name@Company.com\n' 
```

entered in the config as aliases, space separated:

default_reviewers: 'Name1 Name2'
user_aliases:
  Name1: Name.One@Company.com
  Name2: Name.Two@Company.com

Also tried with just having adresses spelled out - this did not work either in version 0.5.3 or version 0.6 (though in the latter I got an unexpected error that aliases were not set, which is what first caused it to fail - which is exactly why I switched to an alias setup).

In addition, I have the following environment variables set:
export DOING_CREATE_PR_CHECKOUT="True"
export DOING_CREATE_ISSUE_MINE="True"

I believe, summarizing, that this is a bug with how (multiple?) reviewers are being parsed.

**Comment by PaulHofma** [comment]
Realized that this issue is due to how bash handles separate arguments to the --reviewer call. For my use case, I could fix this by using environment variables and inputting the reviewers as bash array: 

`export DOING_CREATE_PR_REVIEWERS=("Name.One@Company.com" "Name.Two@Company.com")`

However, this is not in line with the current documentation and also does not work for the config yaml (I found no way to circumvent this issue there).

**Comment by timvink** [comment]
Thanks for reporting. I'll have a look and do some tests myself.

Azure has case-sensitive emailadresses. I got the 'Could not resolve identity' error myself a couple times. It's often due to the emailadres not being properly capitalized. 

At the least, we could catch the error and provide an extra helpful message. Maybe we can even fetch the emails of the top 10 contributors of the projects and display that.

**Comment by PaulHofma** [comment]
Emailadresses are properly capitalized, as far as I can tell (using CamelCase: First.Last@company.com).

**Comment by timvink** [comment]
OK found the cause! It's exactly [this issue](https://github.com/Azure/azure-devops-cli-extension/issues/733). The azure docs are confusing because they do suggest it's a string with spaces.

I just ran a test, and the proper syntax that works is not:

`--reviewers 'Name.One@Company.com Name.Two@Company.com My.Name@Company.com'`

but 

`--reviewers 'Name.One@Company.com' 'Name.Two@Company.com' 'My.Name@Company.com'`

So in `doing` the following changes are required:

- Properly split the reviewers string on `doing create pr` for azure devops
- support both input types in `doing`, either a list of reviewers or a string with space separated values
- Update the docs with examples of multiple reviewers

**Comment by timvink** [comment]
Released in `v0.7`, thanks again for reporting!
